lib/exercism/active_record_citext.rb

```diff
@@ -1,4 +0,0 @@
-# from http://gray.fm/2013/09/17/unknown-oid-with-rails-and-postgresql/
-ActiveRecord::ConnectionAdapters::PostgreSQLAdapter.tap do |klass|
-  klass::OID.register_type('citext', klass::OID::Identity.new)
-end
```

Comment:
Yeah `citext` lets us do very efficient case insensitive lookups on a value (in this case the github username of the user).